packages/core/src/lib/sign-in.ts

```diff
@@ -121,12 +121,15 @@ export const assignRedirectUrlForSocial = async (
 export const signInWithSocial = async (
   ctx: WithUserLogContext<Context>,
   provider: Provider,
-  { connectorId, code }: { connectorId: string; code: string }
+  { connectorId, code, result }: { connectorId: string; code: string; result?: InteractionResults }
 ) => {
   ctx.userLog.connectorId = connectorId;
   ctx.userLog.type = UserLogType.SignInSocial;
 
-  const userInfo = await getUserInfoByConnectorCode(connectorId, code);
+  const userInfo =
+    code === 'session'
```

Comment:
`code` is supposed to be a random string? will it be possible for `code` to be `session`?

Reply:
i'm lost too. will it be better to add `type` or something to explicitly indicate data source? or should we just pass `userInfo` in this function?

Reply:
My fault, forgot to leave a comment here, this will be refactored into seperate route, then the `code` can be removed.

Reply:
cool